asciidoc-confluence-publisher-doc/etc/docs/00-index/06-images.adoc

```diff
@@ -22,6 +22,18 @@ image::../images/frisbee.png[]
 .A nice orange frisbee
 image::../images/frisbee.png[]
 
+Remote (i.e., images with a URL target) block and inline images are also supported.
```

Comment:
Maybe add a section title "Remote Images" (same as "Inline Images") - anyway the structure of the various image types is a bit tricky, as remote vs. local and inline vs. block are orthogonal ...